### PR TITLE
Show profile image on profile page

### LINKED ISSUE #2766: Profile page needs a few things
I think the profile page should have the following things:
- [ ] profile image of the user 
- [x] list of teams the user is a member of

### PR DESCRIPTION
**Pull request by tejasbubane**
Closes #2766 

Looks like this:

screen shot 2016-04-26 at 3 27 57 pm

### CODE REVIEW

**Comment by tejasbubane** [comment]
I ran `compass compile` and don't know why it changed some files like `submission.css`, `footer.css` and `font-awesome.css`. Not sure if I should keep them in the PR.

And one more thing, do we have a `full_name` of the user stored anywhere? Could not see it in `users` table. It would be better to show the `full_name` rather than `user_name` besides that image.
cc:\ @kytrinyx 

github_full_name

File: app/views/user/show.erb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <div class="container">
-  <section class="page-header">
-  <h1><%= profile.username %></h1>
+  <section class="row page-header">
+    <div class="col-md-1 main-profile-img"><img src="https://avatars.githubusercontent.com/u/980783?s=90" /></div>
```
Comment (brettchalupa) [inline app/views/user/show.erb]: @tejasbubane It looks like you hard-coded the image src to your avatar. I think it should be dynamic based on the user that is currently being viewed.

Comment (brettchalupa) [inline app/views/user/show.erb]: @tejasbubane I did some looking into the code, and I think something like:

```
<%= gravatar_tag current_user.avatar_url, size: 90 %>
```

would work ([based on nav/account.erb](https://github.com/exercism/exercism.io/blob/master/app/views/application/nav/account.erb#L3)).

Comment (tejasbubane) [inline app/views/user/show.erb]: Thanks @brettchalupa for pointing that out. I forgot to change the `url`.

**Comment by kytrinyx** [comment]
I'm not storing the full name, just the username. I probably forgot to regenerate the CSS earlier. I'll do that and then rebase this.

**Comment by kytrinyx** [comment]
Sorry that took a while. I've regenerated the CSS and pushed it up to master. Do you want to rebase this, or should I do the merge locally?

**Comment by tejasbubane** [comment]
@kytrinyx rebased

**Comment by kytrinyx** [comment]
Thank you!

File: app/views/user/show.erb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <div class="container">
-  <section class="page-header">
-  <h1><%= profile.username %></h1>
+  <section class="row page-header">
+    <div class="col-md-1 main-profile-img"><%= gravatar_tag profile.current_user.avatar_url, size: 90 %></div>
```
Comment (tachyons) [inline app/views/user/show.erb]: When you visit someone else's profile , you will see your profile image instead of their profile image

Comment (kytrinyx) [inline app/views/user/show.erb]: Thanks for catching that.

Comment (kytrinyx) [inline app/views/user/show.erb]: Deploying a fix now.

Comment (tejasbubane) [inline app/views/user/show.erb]: Thanks @tachyons I missed that.

**Pull request merged**